src/Components/Backoffice/Home/Edit.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { useFormik } from "formik";
+import Button from "@material-ui/core/Button/Button";
+import EditItem from "./EditItem";
+
+const Edit = (props) => {
+  const { homeWelcomeTitle } = props;
+  const [slides, setSlides] = useState([]);
+  const [slidesIds, setSlidesIds] = useState([]);
+
+  useEffect(() => {
+    async function getUser() {
+      try {
+        const response = await axios.get(
+          `http://ongapi.alkemy.org/api/slides/`
+        );
+        setSlides(response.data.data);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+    getUser();
+  }, [slides]);
+
+  const formik = useFormik({
+    initialValues: {
+      homeWelcomeTitle: homeWelcomeTitle,
```

Comment:
if the value has the same name of the object key then, its not required to assign it, you can just make
```
initialValues: { homeWelcomeTitle }
```